packages/nimble-components/build/generate-workers/source/tests/render-worker.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { RenderWorker } from '../render-worker';
+
+describe('RenderWorker', () => {
+    let worker: RenderWorker;
+
+    beforeEach(() => {
+        worker = new RenderWorker();
+    });
+
+    it('emptyMatrix should empty the dieMatrix', () => {
+        worker.dieMatrix = Uint8Array.from([1, 2, 3]);
+        worker.emptyMatrix();
+        expect(worker.dieMatrix.length).toEqual(0);
+    });
+
+    it('updateMatrix should update the dieMatrix correctly', () => {
+        const testData: Iterable<number> = [4, 5, 6];
+        worker.updateMatrix(testData);
+        expect(worker.dieMatrix).toEqual(Uint8Array.from(testData));
+    });
+});
```

Comment:
Are there any tests you can add to inspect `Comlink`'s state and verify these lines?

```ts
const worker = new RenderWorker();
Comlink.expose(worker);
```

Reply:
Discussed offline and there isn't an easy way to test via a unit test but this is covered by integration tests. Closing.